src/components/SkillRating/SkillRatingCard.js

```diff
@@ -91,6 +91,15 @@ export default class SkillRatingCard extends Component {
     }
   };
 
+  roundOffRating = ratingsOverTime => {
+    ratingsOverTime = ratingsOverTime.map(obj => {
+      var curr_obj = obj;
+      curr_obj.rating = parseFloat(curr_obj.rating.toFixed(1));
+      return curr_obj;
```

Comment:
This can be condenced ever further. Just straight away return like this

```
return {...obj, rating: parseFloat(curr_obj.rating.toFixed(1))}
```
This will save you an extra variable.

Reply:
and if you are just returning, you don't even need the **return** keyword.